ocs_ci/deployment/deployment.py

```diff
@@ -995,7 +995,8 @@ def setup_local_storage(storageclass):
             "Creating optional operators CatalogSource and ImageContentSourcePolicy"
         )
         run_cmd(f"oc create -f {optional_operators_yaml.name}")
-
+        # sleep here to start update machineconfigpool status
+        time.sleep(60)
```

Comment:
Will be good if you will print the message mentioned in the comment to the log. WDYT?

Reply:
Sure will add a log here

Reply:
Done